modules/images-create-s2i-build.adoc

```diff
@@ -1,39 +1,27 @@
 // Module included in the following assemblies:
 //
 //* builds/build-strategies.adoc
+// * openshift_images/create-images.adoc
 
 [id="images-create-s2i-build_{context}"]
-= Understanding the S2I build process
+= Understanding the source-to-image build process
 
 The build process consists of the following three fundamental elements, which
 are combined into a final container image:
 
-* sources
-* S2I scripts
-* builder image
-
-During the build process, S2I must place sources and scripts inside the builder
-image. To do so, S2I creates a `tar` file that contains the sources and
-scripts, then streams that file into the builder image. Before executing the
-`assemble` script, S2I un-tars that file and places its contents into the
-location specified by the `io.openshift.s2i.destination`
-label from the builder image, with the default location being the
-`/tmp` directory.
-
-For this process to happen, your image must supply the `tar` archiving utility
-(the `tar` command available in `$PATH`) and the command line interpreter (the
-`/bin/sh` command); this allows your image to use the fastest possible build
-path. If the `tar` or `/bin/sh` command is not available, the `s2i build`
-process is forced to automatically perform an additional container build to put
-both the sources and the scripts inside the image, and only then run the usual
-build.
+* Sources
+* Source-to-image (S2I) scripts
+* Builder image
+
+S2I generates a Dockerfile with the builder image as the first `FROM` instruction. The Dockerfile generated by S2I is then passed to Buildah.
+
+////
+The old workflow was out of date and removed in OSDOCS-999. The Source-To-Image (S2I) README.md on Github is also out of date.
 
 See the following diagram for the basic S2I build workflow:
 
 .Build Workflow
 image::s2i-flow.png[S2I workflow]
 
-Run build's responsibility is to un-tar the sources, scripts and artifacts
-(if such exist) and invoke the `assemble` script. If this is the second run
-(after catching `tar` or `/bin/sh` not found error) it is responsible only
-for invoking `assemble` script, since both scripts and sources are already there.
+Run build's responsibility is to untar the sources, scripts, and artifacts and invoke the `assemble` script. If this is the second run, after catching `tar` or `/bin/sh` not found error, it is only responsible for invoking the `assemble` script, since both scripts and sources are already there.
```

Comment:
According adam's comments: https://github.com/openshift/openshift-docs/pull/27435#discussion_r542396349, do we need this part?

Reply:
You're right! I had it commented out, but I will go ahead and remove it completely.